src/components/ex-live-search-results/ex-live-search-results.scss

```diff
@@ -0,0 +1,101 @@
+.ex-live-search-results {
+  @apply overscroll-contain absolute hidden overflow-y-auto bg-white;
+
+  box-shadow: 0px 3px 45px #0000002c;
+  max-height: calc(100vh - 200px);
+  top: 100%;
+  width: 120%;
+
+  ul.ex-live-search-results__list {
+    @apply ml-0;
+  }
+
+  .ex-search-results-item {
+    @apply block clear-both px-5 pt-4 mt-0 overflow-hidden cursor-pointer;
+
+    &__media {
```

Comment:
Add an .ml-1 to the image or so. So that the text does not stick to the image so tightly.

![image](https://user-images.githubusercontent.com/88790097/141469757-c7f499c5-c5d4-456f-bbe2-98cc271f17be.png)

Reply:
added ml-1